renovate.json

```
@@ -12,6 +12,7 @@
   "branchPrefix": "dependencies/",
   "commitMessagePrefix": "Deps: ",
   "labels": ["dependencies"],
+  "schedule": ["after 9am and before 17pm on Monday"],
```

```suggestion
  "schedule": ["after 9am and before 5pm on Monday"],
```

(or if it can handle 24h format, then `after 9:00 and before 17:00` - but I didn't find anything about this in docs 🤔)

---

Oh shit, my bad 🤦‍♂️  My entire life is in 24h format :-)